keel-titus-plugin/src/main/kotlin/com/netflix/spinnaker/keel/api/titus/cluster/TitusClusterHandler.kt

```diff
@@ -113,6 +120,48 @@ class TitusClusterHandler(
         .map { it.await() }
     }
 
+  override suspend fun export(exportable: Exportable): SubmittedResource<TitusClusterSpec> {
+    // TODO[GY] : remove unchanged/default fields from response (across all export responses)
+    val serverGroups = cloudDriverService.getServerGroups(
+      exportable.account,
+      exportable.moniker,
+      exportable.regions,
+      exportable.serviceAccount
+    ).byRegion()
+
+    if (serverGroups.isEmpty()) {
+      throw ResourceNotFound("Could not find cluster: ${exportable.moniker.name} " +
+        "in account: ${exportable.account} for export")
+    }
+
+    val base = serverGroups.values.first()
+
+    val locations = SimpleLocations(
+      account = exportable.account,
+      regions = (exportable.regions.map {
```

Comment:
This field (`exportable.regions`) is actually a list of all the regions that aren't deprecated in the account I think. Right now, when you export you get a spec with all the regions defined even if you export a resource in only one region. I think that `regions` here should be `serverGroups.keys` so that we get the actual regions.

Reply:
Fixed.